distribution/tools/keystore-cli/src/main/java/org/elasticsearch/common/settings/AddStringKeyStoreCommand.java

```diff
@@ -42,13 +46,13 @@
     private final OptionSpec<String> arguments;
 
     AddStringKeyStoreCommand() {
-        super("Add a string setting to the keystore", false);
-        this.stdinOption = parser.acceptsAll(Arrays.asList("x", "stdin"), "Read setting value from stdin");
+        super("Add string settings to the keystore", false);
```

Comment:
Would it be worth updating the equivalent text in `RemoveSettingKeyStoreCommand`? It still talks about removing "a setting" from the keystore, but it already supports multiple settings.

Reply:
I will handle this in a follow up.

Reply:
I opened #54244.